gensim/models/fasttext.py

```diff
@@ -704,6 +708,14 @@ def train(self, sentences=None, corpus_file=None, total_examples=None, total_wor
             >>> model.train(sentences, total_examples=model.corpus_count, epochs=model.epochs)
 
         """
+        cant_train = hasattr(self.trainables, 'syn1neg') and self.trainables.syn1neg is None
```

Comment:
stupid question: what if self.trainables does not have `syn1neg` attr at all, so can model train ?

Reply:
Yes. I don't see any other code that sets syn1neg to None. So, the new code uses that value to mean "cannot continue training".

If trainables does not have syn1neg at all, it is possible to _start_ training.